CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+- Rename cred explorer table columns (#680)
```

Comment:
Should this go at the bottom, for consistency? We can `tac` it when we
commit a release.